storage/backup/backup-cli/src/metadata/cache.rs

```diff
@@ -118,11 +120,14 @@ pub async fn sync_and_load(
                 &mut OpenOptions::new()
                     .write(true)
                     .create_new(true)
-                    .open(&local_file)
+                    .open(&local_tmp_file)
                     .await
                     .err_notes(&local_file)?,
             )
             .await?;
+            // rename to target file only if successful; stale tmp file caused by failure will be
```

Comment:
could also use tmpfs?

Reply:
Transferring the file from tmpfs to the target folder is slower and tends more to fail than a renaming?